Trigger/data/tprDeP_highP_stopTarg/main_tprDeP_highP_stopTarg_tcFilter.fcl

```diff
@@ -1,2 +1,2 @@
 physics.filters.tprDePHighPStopTargTCFilter.requireCaloCluster : false
-physics.filters.tprDePHighPStopTargTCFilter.minNStrawHits : 10.0
```

Comment:
"minNStrawHits" is an int.  Please remove the decimal point.  This applies to all the entries

Reply:
done